src/main/java/ReadyGame.java

```diff
@@ -0,0 +1,62 @@
+import java.util.*;
+
+public class ReadyGame {
+
+    public String inputUserNumber() {
+        String userNumber;
+        Scanner scanner;
+
+        scanner = new Scanner(System.in);
+        userNumber = scanner.nextLine();
```

Comment:
지역 변수의 초기화는 아래와 같은 방식으로 해주시면 좋을 것 같아요 :)

```suggestion
        String scanner = new Scanner(System.in);
        Scanner userNumber = scanner.nextLine();
```